Bio/Seq.py

```diff
@@ -3213,8 +3233,6 @@ def defined(self):
 
         Zero-length sequences are always considered to be defined.
         """
-        if self._length == self._data_length:
-            return True
```

Comment:
Again, why remove this?

Reply:
Because it never happens (and therefore `self._data_length` does not exist any more).

Reply:
Why can't ``_PartiallyDefinedSequenceData`` be length zero? It has a ``__bytes__`` method which explicitly considers the case ``self._length == 0``

Reply:
(My original comment here was too vague - my point is shouldn't this property return ``True`` for zero length?)

Reply:
Zero length will never happen.

Reply:
The test in the `__bytes__` method has now been removed.